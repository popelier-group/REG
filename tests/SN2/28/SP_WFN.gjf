# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.43329900   -0.00002700    0.00010000
 H                 -1.10606000   -0.87518400   -0.54540900
 H                 -1.10603200    0.90999000   -0.48500900
 H                 -1.10618500   -0.03490500    1.03080400
 Br                 1.71864800    0.00000700   -0.00000900
 Cl                -3.29118600   -0.00000400   -0.00002400

frame.wfn
