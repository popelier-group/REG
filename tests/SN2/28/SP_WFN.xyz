
6

C        1.306459    0.000804    0.030456
H        1.000018   -0.544871   -0.852096
H        0.958318   -0.484494    0.932592
H        0.979689    1.031331   -0.012043
Br       -1.844630   -0.001013   -0.043029
Cl        3.163840    0.001687    0.073814
