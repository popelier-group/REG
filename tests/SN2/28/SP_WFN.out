 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.28/Gau-175174.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.28/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    175175.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.4333   -0.00003   0.0001 
 H                    -1.10606  -0.87518  -0.54541 
 H                    -1.10603   0.90999  -0.48501 
 H                    -1.10618  -0.03491   1.0308 
 Br                    1.71865   0.00001  -0.00001 
 Cl                   -3.29119  0.        -0.00002 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.433299   -0.000027    0.000100
      2          1           0       -1.106060   -0.875184   -0.545409
      3          1           0       -1.106032    0.909990   -0.485009
      4          1           0       -1.106185   -0.034905    1.030804
      5         35           0        1.718648    0.000007   -0.000009
      6         17           0       -3.291186   -0.000004   -0.000024
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.081927   0.000000
     3  H    1.081927   1.786196   0.000000
     4  H    1.081929   1.786202   1.786201   0.000000
     5  Br   3.151947   3.007058   3.007010   3.007237   0.000000
     6  Cl   1.857887   2.416229   2.416236   2.416206   5.009834
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        1.306459    0.000804    0.030456
      2          1           0        1.000018   -0.544871   -0.852096
      3          1           0        0.958318   -0.484494    0.932592
      4          1           0        0.979689    1.031331   -0.012043
      5         35           0       -1.844630   -0.001013   -0.043029
      6         17           0        3.163840    0.001687    0.073814
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.1708822           0.7873869           0.7873868
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       166.4972224200 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.71D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40249230     A.U. after   11 cycles
            NFock= 11  Conv=0.74D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55490-101.41125 -62.20791 -56.02572 -56.02565
 Alpha  occ. eigenvalues --  -56.02565 -10.08428  -9.32550  -8.41662  -7.08924
 Alpha  occ. eigenvalues --   -7.08073  -7.08073  -6.23969  -6.23934  -6.23934
 Alpha  occ. eigenvalues --   -2.35207  -2.35196  -2.35196  -2.35168  -2.35168
 Alpha  occ. eigenvalues --   -0.71074  -0.54404  -0.48811  -0.30828  -0.30828
 Alpha  occ. eigenvalues --   -0.27345  -0.17680  -0.17680  -0.05354  -0.05234
 Alpha  occ. eigenvalues --   -0.05234
 Alpha virt. eigenvalues --    0.10899   0.12253   0.14798   0.14798   0.17468
 Alpha virt. eigenvalues --    0.18267   0.19535   0.19535   0.20677   0.20677
 Alpha virt. eigenvalues --    0.21665   0.22395   0.22395   0.24787   0.28226
 Alpha virt. eigenvalues --    0.30649   0.30649   0.36581   0.40403   0.53433
 Alpha virt. eigenvalues --    0.53433   0.55252   0.58494   0.58494   0.60701
 Alpha virt. eigenvalues --    0.62068   0.62069   0.64395   0.76437   0.76437
 Alpha virt. eigenvalues --    0.79485   0.84290   0.87934   0.87934   0.89250
 Alpha virt. eigenvalues --    0.89250   0.95083   0.96631   0.96631   1.00949
 Alpha virt. eigenvalues --    1.00950   1.03136   1.07909   1.26177   1.26177
 Alpha virt. eigenvalues --    1.26652   1.56845   1.67520   1.67520   1.84880
 Alpha virt. eigenvalues --    1.84880   1.86873   2.09703   2.21506   2.21508
 Alpha virt. eigenvalues --    2.27288   2.27288   2.28863   2.45509   2.45509
 Alpha virt. eigenvalues --    2.56371   2.59289   2.63964   2.63964   2.66893
 Alpha virt. eigenvalues --    2.66893   3.10349   3.28913   3.87012   3.87012
 Alpha virt. eigenvalues --    4.39663   4.39665   4.40377   4.40378   4.51088
 Alpha virt. eigenvalues --    6.76173   7.80379   7.80379   7.85683   9.83930
 Alpha virt. eigenvalues --   24.03131  25.98586  25.98586  26.48865  48.07567
 Alpha virt. eigenvalues --  215.82501 290.01214 290.01214 290.060541020.83287
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.645678   0.402736   0.402736   0.402740  -0.054913  -0.147255
     2  H    0.402736   0.519191  -0.038140  -0.038142   0.001603  -0.039892
     3  H    0.402736  -0.038140   0.519184  -0.038141   0.001602  -0.039888
     4  H    0.402740  -0.038142  -0.038141   0.519217   0.001606  -0.039909
     5  Br  -0.054913   0.001603   0.001602   0.001606  35.922251   0.052357
     6  Cl  -0.147255  -0.039892  -0.039888  -0.039909   0.052357  17.216279
 Mulliken charges:
               1
     1  C   -0.651722
     2  H    0.192643
     3  H    0.192647
     4  H    0.192629
     5  Br  -0.924506
     6  Cl  -0.001692
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.073802
     5  Br  -0.924506
     6  Cl  -0.001692
 Electronic spatial extent (au):  <R**2>=           1214.3888
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              4.5501    Y=              0.0027    Z=              0.1061  Tot=              4.5513
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -75.9777   YY=            -44.4468   ZZ=            -44.4639
   XY=             -0.0169   XZ=             -0.7356   YZ=             -0.0004
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -21.0149   YY=             10.5160   ZZ=             10.4989
   XY=             -0.0169   XZ=             -0.7356   YZ=             -0.0004
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            -45.5521  YYY=              0.8496  ZZZ=             -0.5149  XYY=             -8.6060
  XXY=             -0.0155  XXZ=             -0.6611  XZZ=             -8.6201  YZZ=             -0.8676
  YYZ=             -0.2887  XYZ=              0.0199
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1629.3269 YYYY=            -76.3366 ZZZZ=            -77.0942 XXXY=             -0.4649
 XXXZ=            -19.8684 YYYX=              0.3819 YYYZ=              0.0088 ZZZX=            -16.2841
 ZZZY=             -0.0621 XXYY=           -259.2046 XXZZ=           -259.4194 YYZZ=            -25.5780
 XXYZ=              0.0308 YYXZ=             -5.5308 ZZXY=             -0.8883
 N-N= 1.664972224200D+02 E-N=-7.686331092663D+03  KE= 3.070030343184D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE719\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.433299,-0.000027,0.0001\H,0,-1.10606,-0.875184,-0.545409\H,0,-1
 .106032,0.90999,-0.485009\H,0,-1.106185,-0.034905,1.030804\Br,0,1.7186
 48,0.000007,-0.000009\Cl,0,-3.291186,-0.000004,-0.000024\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.4024923\RMSD=7.383e-09\Dipole=-1.79063
 42,-0.0000243,0.0000839\Quadrupole=-15.6368323,7.8184274,7.8184049,-0.
 0000272,-0.0001521,0.0000011\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 OLD HORSE! OLD HORSE! WHAT BROUGHT YOU HERE?
 FROM SACARAP' TO PORTLAND PIER
 I'VE CARTED STONE THIS MANY A YEAR;
 TILL, KILLED BY BLOWS AND SORE ABUSE,
 THEY SALTED ME DOWN FOR SAILORS' USE.
 THE SAILORS THEY DO ME DESPISE;
 THEY TURN ME OVER AND DAMN MY EYES;
 CUT OFF MY MEAT, AND SCRAPE MY BONES,
 AND PITCH ME OVER TO DAVY JONES.

    SAILORS' COMPLAINT ABOUT THE BEEF
    SERVED ON SHIPBOARD, CIRCA 1835.
 Job cpu time:       0 days  0 hours  0 minutes 15.7 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  9.0 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:56:32 2020.
